### PR TITLE
Feature/live site

### PR DESCRIPTION
**Pull request by chaung844**
Added live site (excluded for now)
Added redirect to live site if _siteConfigData.timeline === "live"_ (config.json)

### CODE REVIEW

File: package-lock.json
Comment (menamerai) [inline package-lock.json:1]: Remove this file. We use yarn, so `package-lock.json` should not be here.

File: src/components/Hero.astro [hunk 1]
```diff
@@ -7,6 +7,7 @@ const { siteState } = Astro.props; // for updating content depending on the stag
 let title = "Revolution".toUpperCase();
 let date = "    February 24-25    ".toUpperCase(); // Keep the space in order to match with the RevolutionUC Text
```
Comment (menamerai) [inline src/components/Hero.astro:8]: Can you hide this for the livesite? We don't need the date when the hackathon is live.

File: src/pages/_live.astro
Comment (menamerai) [inline src/pages/_live.astro:1]: Can you add some "quick links" that hackers can refer to on the livesite? It'll be these links:

image

**Review by menamerai — CHANGES_REQUESTED** [review]

**Comment by menamerai** [comment]
Some pushes were made sine your last update, you might need to rebase or merge

File: src/pages/_live.astro
Comment (chaung844) [reply src/pages/_live.astro:1]: not sure where to place them 
Screenshot 2024-02-23 at 12 59 21 AM

**Review by menamerai — APPROVED** [review]

**Pull request merged**